src/kernel/src/syscalls/mod.rs

```diff
@@ -243,6 +243,9 @@ impl Syscalls {
                     todo!("regmgr_call with matched key = {key:#x}");
                 }
             }
+            // Pass default value for any values that aren't given a unique case
+            0x27 | 0x38..=u32::MAX => -2146631143,
+            // Any missing cases TODO
```

Comment:
```c
    case 0x38:
        if (DAT_ffffffffa2830830 == 0) goto LAB_ffffffffa08df0f3;
        if ((DAT_ffffffffa2830834 != 0) && (DAT_ffffffffa2830800 != 1)) goto LAB_ffffffffa08df0dd;
        key = FUN_ffffffffa08d04d0();
        goto LAB_ffffffffa08df4bf;
```

Reply:
Stop going one by one because the FW leak only goes to 36 XD

Reply:
If you mean the one that you send it to me it is not a source leak but an old binary leak that was decompiled by IDA Pro. You should follow the most recent one that you have, which is 9.0; and use the leaked one as a supplement.

Reply:
Ah, I see 0x39 is the highest so far.